doc/representation-for-multiple-content-types.md

```diff
@@ -0,0 +1,81 @@
+## Decision Record: Representation for Multiple Content Types
```

Comment:
This file looks great.  Would it be better in doc/arch/ to live with the existing ADR that's there?